12321

```diff
@@ -0,0 +1,18 @@
+#include<stdio.h>
```

Comment:
@Harman-kamboj - name the directories wisely, enter the sample input and output in the comments and if possible create a README.md file.